ecosystem/sep-0024.md

```diff
@@ -1067,29 +1067,30 @@ Each object in the `transactions` array should have the following fields:
+| `amount_in`               | string              | Amount received by anchor at start of transaction as a string with up to 7 decimals. Excludes any fees charged before the anchor received the funds.                                                                                                                                                                              |
+| `amount_in_asset`         | string              | (optional) The asset received or to be received by the Anchor. Must be present if the deposit/withdraw was made using non-equivalent assets. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). See the [Asset Exchanges](#asset-exchanges) section for more information.        |
+| `amount_out`              | string              | Amount sent by anchor to user at end of transaction as a string with up to 7 decimals. Excludes amount converted to XLM to fund account and any external fees.                                                                                                                                                                    |
+| `amount_out_asset`        | string              | (optional) The asset delivered or to be delivered to the user. Must be present if the deposit/withdraw was made using non-equivalent assets. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). See the [Asset Exchanges](#asset-exchanges) section for more information.        |
+| `amount_fee`              | string              | (**deprecated**) Amount of fee charged by anchor. This field is deprecated in favor of the `fee_details` field.                                                                                                                                                                                                                   |
+| `amount_fee_asset`        | string              | (**deprecated**, optional) The asset in which fees are calculated in. Must be present if the deposit/withdraw was made using non-equivalent assets. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). See the [Asset Exchanges](#asset-exchanges) section for more information. |
+| `fee_details`             | object              | Description of fee charged by the anchor. The schema for this object is defined in the [Fee Details Object Schema](#fee-details-object-schema) section below.                                                                                                                                                                     |
```

Comment:
Lets add a note here that `fee_details` should match the referenced quote's `fee` object.